wallabag_kindle_consumer/config.py

```diff
@@ -24,7 +24,7 @@ class Config:
     interface_port: int = 8080
 
     @staticmethod
-    def from_file(filename):
+    def from_file(filename: str) -> "Config" | None:
```

Comment:
Great job!
I only have a doubt: What is the difference between `"Config" | None` and `Optional["Config"]` as type hint?  I tested this change, and as a result, I encountered the following error:
``` bash
wallabag-kindle-consumer    | Traceback (most recent call last):
wallabag-kindle-consumer    |   File "/app/./service.py", line 12, in <module>
wallabag-kindle-consumer    |     from wallabag_kindle_consumer import models
wallabag-kindle-consumer    |   File "/app/wallabag_kindle_consumer/models.py", line 7, in <module>
wallabag-kindle-consumer    |     from wallabag_kindle_consumer.config import Config
wallabag-kindle-consumer    |   File "/app/wallabag_kindle_consumer/config.py", line 9, in <module>
wallabag-kindle-consumer    |     class Config:
wallabag-kindle-consumer    |   File "/app/wallabag_kindle_consumer/config.py", line 27, in Config
wallabag-kindle-consumer    |     def from_file(filename: str) -> "Config" | None:
wallabag-kindle-consumer    |                                     ~~~~~~~~~^~~~~~
wallabag-kindle-consumer    | TypeError: unsupported operand type(s) for |: 'str' and 'NoneType'
```
However, employing `Optional["Config"]` resolved the issue

Reply:
Good catch! That should have been `"Config | None"` for it to work.

With the latest changes to the Configuration class, we can now use `from __future__ import annotations` ([source](https://peps.python.org/pep-0563/)) instead, so we don't need to use strings for type hints at all!